File: app/backend/storage/code_execution/f630200e-a48e-4ef4-9dce-14872556ee76/code.c
#include <stdio.h>
#include <string.h>

int main() {
    int n;
    scanf("%d", &n);

    char best_name[101];  // Pretpostavljamo da ime ima najviše 100 karaktera
    double best_avg = -1.0;

    for (int i = 0; i < n; i++) {
        char name[101];
        double jump1, jump2;

        scanf("%s", name);
        scanf("%lf", &jump1);
        scanf("%lf", &jump2);

        double avg = (jump1 + jump2) / 2.0;
        if (avg > best_avg) {
            best_avg = avg;
            strcpy(best_name, name);
        }
    }

    printf("%s\n", best_name);
    return 0;
}

File: app/backend/storage/code_execution/18d7e888-950d-4bdb-ae86-d32429012437/code.py
def best_long_jump_participant():
    n = int(input().strip())  # Broj polaznika
    best_name = ""
    best_avg = -1

    for _ in range(n):
        name = input().strip()  # Ime polaznika
        jump1 = float(input().strip())  # Prvi skok
        jump2 = float(input().strip())  # Drugi skok
        avg = (jump1 + jump2) / 2

        if avg > best_avg:
            best_avg = avg
            best_name = name

    print(best_name)

# Pokretanje funkcije
best_long_jump_participant()